content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -867,6 +873,53 @@ is disabled, `.spec.completions` is immutable.
 Use cases for elastic Indexed Jobs include batch workloads which require 
 scaling an indexed Job, such as MPI, Horovord, Ray, and PyTorch training jobs.
 
+### Delayed creation of replacement pods
+
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+{{< note >}}
+You can only set `podReplacementPolicy` on Jobs if you enable the `JobPodReplacementPolicy` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+{{< /note >}}
+
+By default, the Job controller recreates Pods as soon they either fail or are terminating (have a deletion timestamp).
```

Comment:
Does it make sense to make this more detailed here? That it recreates on success as well..

Reply:
Why would Job recreate a Pod on success?

Reply:
https://github.com/kubernetes/website/pull/41745#discussion_r1290132049